fearless/Modules/AccountImport/AccountImportViewController.swift

```diff
@@ -214,8 +212,12 @@ final class AccountImportViewController: UIViewController {
             isEnabled = isEnabled && !(passwordTextField.text?.isEmpty ?? true)
         }
 
-        if let viewModel = derivationPathModel, viewModel.inputHandler.required {
-            isEnabled = isEnabled && !(derivationPathField.text?.isEmpty ?? true)
+        if let viewModel = substrateDerivationPathModel, viewModel.inputHandler.required {
+            isEnabled = isEnabled && !(substrateDerivationPathField.text?.isEmpty ?? true)
```

Comment:
why not &= ?

Reply:
&= is a bitwise operation and can only be applied to integer types. &&= which is what we need doesn't actually exist because of short circuiting.